test/prow/fuse_recovery.py

```diff
@@ -0,0 +1,307 @@
+    api = client.CustomObjectsApi()
+    my_dataset = {
+        "apiVersion": "data.fluid.io/v1alpha1",
+        "kind": "Dataset",
+        "metadata": {"name": "hbase"},
+        "spec": {
+            "mounts": [{"mountPoint": "https://mirrors.bit.edu.cn/apache/zookeeper/stable/", "name": "hbase"}]
+        }
+    }
+
+    my_alluxioruntime = {
+        "apiVersion": "data.fluid.io/v1alpha1",
+        "kind": "AlluxioRuntime",
+        "metadata": {"name": "hbase"},
+        "spec": {
+            "replicas": 1,
+            "tieredstore": {
+                "levels": [{
+                    "mediumtype": "MEM",
+                    "path": "/dev/shm",
+                    "quota": "2Gi",
+                    "high": "0.95",
+                    "low": "0.7"
+                }]
+            }
+        }
+    }
+
+    api.create_namespaced_custom_object(
+        group="data.fluid.io",
+        version="v1alpha1",
+        namespace=namespace,
+        plural="datasets",
+        body=my_dataset,
+    )
+    print("Created dataset.")
+    api.create_namespaced_custom_object(
+        group="data.fluid.io",
+        version="v1alpha1",
+        namespace=namespace,
+        plural="alluxioruntimes",
+        body=my_alluxioruntime
+    )
+    print("Created alluxioruntime.")
+
+
+def checkDatasetBound():
+    api = client.CustomObjectsApi()
+    while True:
+        resource = api.get_namespaced_custom_object(
+            group="data.fluid.io",
+            version="v1alpha1",
+            name="hbase",
+            namespace=namespace,
+            plural="datasets"
+        )
+        print(resource)
+        if "status" in resource:
+            if "phase" in resource["status"]:
+                if resource["status"]["phase"] == "Bound":
+                    return True
+        time.sleep(1)
+
+
+def checkFuseRecovered():
+    api = client.CoreV1Api()
+    while True:
+        items = api.list_namespaced_event(namespace=namespace).items
+        for item in items:
+            if item.message.__contains__("Fuse recover"):
+                print("Fuse Recovered.")
+                return True
+        print("Fuse not Recovered.")
+        time.sleep(1)
+
+
+def checkVolumeResourcesReady():
+    while True:
+        try:
+            client.CoreV1Api().read_persistent_volume(name=namespace + "-hbase")
+        except client.exceptions.ApiException as e:
+            if e.status == 404:
+                time.sleep(1)
+                continue
+        try:
+            client.CoreV1Api().read_namespaced_persistent_volume_claim(name="hbase", namespace=namespace)
+        except client.exceptions.ApiException as e:
+            if e.status == 404:
+                time.sleep(1)
+                continue
+        print("PersistentVolume & PersistentVolumeClaim Ready.")
+        break
+
+
+def enableInjection():
+    api = client.CoreV1Api()
+    body = {'metadata': {
+        'labels': {
+            'fluid.io/enable-injection': 'true'
```

Comment:
In the latest  Fluid version, no need to add `fluid.io/enable-injection=true` label on namespace.

Reply:
I referred to some old docs, I'll delete it.